src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -554,15 +593,18 @@ void Pane::_CloseChild(const bool closeFirst)
         // re-attach the TermControl to our Grid.
         _firstChild->_root.Children().Clear();
         _secondChild->_root.Children().Clear();
+        _firstChild->_border.Child(nullptr);
+        _secondChild->_border.Child(nullptr);

```

Comment:
This is not a bad idea IMO.

Reply:
Okay so looking into that one, that's not as bad. This case here is us removing the controls from the child's UI tree, _because we're taking ownership of it_. This is happening because one child closed, and we need to absorb the remaining child's state into ourselves.

When we first split, we do remove our own border, in order to add the two children directly to our root grid. Hypothetically, we could leave that border around always, but we'd need to add another grid to the tree always (since a Border can only ever have one child)